.github/workflows/tsc.yml

```
@@ -0,0 +1,26 @@
+name: Type Check
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+
+      - name: Setup Bun
+        uses: antongolub/action-setup-bun@v1
+        with:
+          cache: true
+          cache-bin: true
+
+      - name: Install
+        run: bun install
+
+      - name: tsc
```

```suggestion
      - name: Type Check
```

For consistency with the others.